Automattic/sensei
# Adjust course theme spacings and alignments

Resolves Automattic/sensei#6834 

## Proposed Changes
* Move some opinionated style from the button.scss file to the file learning-mode-compat.scss
* Update course theme styles to follow the new designs

## Testing Instructions
1. Go to the Branch
2. `npm run start`
3. Active the "Automattic/Course" theme at least version 1.2.4
4.  Check if the layout matches the design files

## Extra Notes
There are opportunities to refactor the file using SCSS selector, but to avoid conflicts with other cards, I avoid doing it now. 


## Pre-Merge Checklist
